packages/server/lambdas/alfresco/upload-file.ts

```diff
@@ -62,6 +63,7 @@ export async function handleRequest(event: ALBEvent): Promise<ALBResult | undefi
     const requestOptions = (await fileRequestBuilder(event, headers)) as RequestInit;
 
     const result = await postFile(requestOptions, categoryData.alfrescoFolder);
+    log.info(user, `uploading file ${result?.entry.name} to ${categoryData.alfrescoFolder}`);
```

Comment:
Split this log into two:  First before the validate user, where you log user and alfrescoFolder. Second here where you log user and file name (folder is also fine to have here). This second should also say "Uploaded", as the upload has already been done at this point.

Just a side note that in alfresco search the log.info is after the validation, which should be fixed. If you can do it here as well (no need to move the debugs), that'd be swell.